LondonBritishLibrary/orient/BLorient3716b.xml

```diff
@@ -0,0 +1,185 @@
+                            <title ref="LIT5888AsmatPrayer">Prayer against bāryā, Satan, and devils</title>
+                            <incipit xml:lang="gez">ድርሱ፡ በስመ፡ አልፋ፡ ፃዕ፡ ፃዕ፡ ፃዕ፡ ወወጺአከ፡ ኢትግባእ፡ ይቤለከ፡ እግዚአብሔር፡ <gap reason="ellipsis"/> አንተ፡ ቡዳ፡ 
+                                ወአንተ፡ ባርያ፡ ወአንተ፡ ሰይጣን፡ ወአንተ፡ ጋኔን፡ ዘትትሜሰሉ፡ በብዙኅ፡ ጾታ፡ አህያ፡ ሺራህያ፡ ይመራኤል፡ ሀያል፡</incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i3">
+                            <title ref="LIT5696MagicPrayer"/>
+                            <incipit xml:lang="gez"><gap reason="ellipsis"/>አአምን፡ በአብ፡ አአምን፡ በወልድ፡ አአምን፡ በመንፈስ፡ ቅዱስ፡ ከልለኒ፡ በወልታ፡ ርድአኒ፡ ወህፅረኒ፡ 
+                                በሐፁረ፡ መስቀልከ፡ ኦአምላኪያ፡ ወመድኀንየ፡ መሐረኒ፡ ወተሠሃለኒ፡ ሊተ፡ ለገብረ፡ እግዚአብሔር፡</incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i4">
+                            <title ref="LIT7049Aqweyasat"/>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ብረር፡ ወከመ፡ ሰምዕ፡ ተመኃው፡ ይቤሉከ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ አንተ፡ ኵሎሙ፡
+                                ወመላኪሆሙ፡ ለአቍያጻት፡ አንተ፡ ባርያ፡ ጸሊም፡ ጋኔን፡ ህሡም፡ ኢትልክኑ፡ ንፍሱ፡ ወሥጋሁ፡ ለገብረ፡ እግዚአብሔር፡ </incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i5">
+                            <title ref="LIT4620Prayer"/>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ጸሎት፡ በእንተ፡ ነሀብት፡ እማሁማ፡ ወያኑራሃሸም፡ <gap reason="ellipsis"/>
+                                አድኅኖ፡ እምነሀብት፡ ለገብረ፡ እግዚአብሔር፡ </incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i6">
+                            <title ref="LIT7046PrayerColic"/>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ጸሎት፡ በእንተ፡ ሕማመ፡ ቍርፀት፡ እንዘ፡ ይትዋነይ፡ በውስተ፡ ማኅፀን፡ እንዘ፡ ይበጽብጽ፡ ከርሠ፡
+                                ወይፈቅል፡ ቍልፈተ፡ ወይደጕጽ፡ ገበዋተ፡ ዝንቱ፡ ህማመ፡ ቍርፀት፡</incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i7">
+                            <title ref="LIT7042PrayerMichael"/>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ጸሎት፡ በእንተ፡ ሌጌዎን፡ ርኩስ፡ ዘይሰልብ፡ ልብ፡ ሰብእ፡ ወያጸልም፡ አእይንተ፡ ወይመጽእ፡
+                                ከመ፡ ጽላሎት፡</incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i8">
+                            <title ref="LIT6989PPFear">Prayer against nocturnal terror</title>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ጸሎተ፡ ድንጋፄ፡ ክርስቶስ፡ ባርከኒ፡ ወአድኅነኒ፡ እምድንጋፄ፡ ወእምፍርሃተ፡ 
+                                <sic resp="PRS8999Strelcyn">ለ</sic>ሊት፡</incipit>
```

Comment:
Why? I have seen it in Strelcyns catalogue and not in the manuscript. Maybe it is easier to silently emend it to ሌሊት፡ or to keep ለሊት፡.

Reply:
If the silent presumption is that the description is based on Strelcyn, then I think it would be ok to have "sic" in your (or my etc.) name, but maybe there is a better solution :)